crates/plugins/mediator-coordination/src/model/stateful/coord.rs

```diff
@@ -279,22 +258,21 @@ pub struct KeylistBody {
 
 /// Keylist entry for a specific key.
 #[derive(Debug, Serialize, Deserialize, Clone, PartialEq, Eq)]
-#[serde(rename_all = "snake_case")]
 pub struct KeylistEntry {
     /// Registered DID
     pub recipient_did: String,
 }
 
 /// Pagination details for a keylist query.
 #[derive(Debug, Serialize, Deserialize, Clone, PartialEq, Eq)]
-#[serde(rename_all = "snake_case")]
 pub struct KeylistPagination {
     pub count: i32,
     pub offset: i32,
     pub remaining: i32,
 }
+
+/// Message to grant a routing DID to a mediator.
 #[derive(Debug, Serialize, Deserialize, Clone, Default)]
-#[serde(rename_all = "snake_case")]
 pub struct MediationGrantBody {
     pub routing_did: String,
 }
```

Comment:
This does not sound valid a description to me. The mediator, as the message sender, does not grant a routing DID to itself. The grant message is instead an agreement to mediate a requesting agent's messages.

Reply:
I don’t know how this comment appeared here, maybe a dumb copy paste. I will fix that.